ui/page/search/index.js

```diff
@@ -26,13 +27,15 @@ const select = (state, props) => {
     showMature === false ? { nsfw: false, isBackgroundSearch: false } : { isBackgroundSearch: false }
   )(state);
   const uris = makeSelectSearchUris(query)(state);
+  const isQueryMaxed = makeSelectIsSearchQueryMaxed(query)(state);
 
   return {
     isSearching: selectIsSearching(state),
     showNsfw: showMature,
     uris: uris,
     isAuthenticated: selectUserVerifiedEmail(state),
     searchOptions: selectSearchOptions(state),
+    isQueryMaxed: isQueryMaxed,
```

Comment:
maybe something like `hasReachedMaxResultsLength`?

Reply:
Sounds good.  Also, I'll rebase to fix the conflict